pkgs/development/tools/asn1ate/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python, pythonPackages, buildPythonPackage, self }:
+
+let
+  pname = "asn1ate";
+  owner = "kimgr";
+in
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  version = "c56104e8912400135509b584d84423ee05a5af6b";
```

Comment:
Please check the latest version of the coding conventions in the Nixpkgs manual. When not using a release the version needs to be a date, and the suffix `-unstable` is added to the name. So e.g. `asn1ate-unstable-2016-08-10`.